include/mbgl/style/expression/literal.hpp

```diff
@@ -12,15 +12,9 @@ namespace expression {
 
 class Literal : public Expression {
 public:
-    Literal(Value value_)
-        : Expression(Kind::Literal, typeOf(value_))
-        , value(value_)
-    {}
-    
-    Literal(type::Array type_, std::vector<Value> value_)
-        : Expression(Kind::Literal, type_)
-        , value(value_)
-    {}
+    Literal(const Value& value_) : Expression(Kind::Literal, typeOf(value_)), value(value_) {}
+
+    Literal(const type::Array& type_, std::vector<Value> value_) : Expression(Kind::Literal, type_), value(value_) {}
```

Comment:
clang-tidy is strange :D type::Array is much lighter than vector and `std::vector<Value> value_` is still copied.

`Expression(Kind::Literal, std::move(type_)), value(std::move(value_))`